srcpkgs/hunspell-ro_RO/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+create_wrksrc=yes
+hostmakedepends="unzip"
```

Comment:
Why did you add this? We use `bsdtar` for zip files.

And I believe it was working without `create_wrksrc` previously?